apiserver/facades/controller/agenttools/agenttools.go

```diff
@@ -125,7 +125,7 @@ func updateToolsAvailability(modelGetter ModelGetter, newEnviron newEnvironFunc,
 		return errors.Annotate(err, "cannot get latest version")
 	}
 	if ver == version.Zero {
-		logger.Debugf("tools lookup returned version Zero, this should only happen during bootstrap.")
+		logger.Debugf("agent binaries lookup returned version Zero, this should only happen during bootstrap.")
```

Comment:
lookup of agent binaries returned ...

That said, This is at Debug level, which means nobody ever sees it, so we should decide if it really is an error, or if we should just drop it entirely.
I'd probably go with the latter.

Reply:
I'll keep because someone somewhere thought it would be useful if things went wrong.... but I will try to make it more .... English, for the lack of a better word :)